diktat-rules/pom.xml

```diff
@@ -17,6 +17,11 @@
     </properties>
 
     <dependencies>
+        <dependency>
+            <groupId>com.squareup</groupId>
+            <artifactId>kotlinpoet</artifactId>
+            <version>1.6.0</version>
```

Comment:
Please declare dependency version in root pom's `dependencyManagement`. It's good practice to keep all versions in single place.